pkg/account/api/api_key_test.go

```diff
@@ -481,19 +487,54 @@ func TestGetAPIKeyMySQL(t *testing.T) {
 					},
 				}, nil)
 			},
-			req:      &accountproto.GetAPIKeyRequest{Id: "id"},
-			expected: nil,
+			req: &accountproto.GetAPIKeyRequest{Id: "id"},
+			getExpectedErr: func(localizer locale.Localizer) error {
+				return nil
+			},
+		},
+		{
+			desc:    "success with viewer account",
```

Comment:
We need to check if the permission error is returned properly. So please use the viewer role for all cases and add an error case with the unassigned role.

Reply:
@kentakozuka 

> We need to check if the permission error is returned properly. So please use the viewer role for all cases and add an error case with the unassigned role.

Please check whether the test case below matches your findings.

```
		{
			desc:    "errPermissionDenied",
			context: createContextWithDefaultToken(t, false),
			setup: func(s *AccountService) {
				s.accountStorage.(*accstoragemock.MockAccountStorage).EXPECT().GetAccountV2ByEnvironmentID(
					gomock.Any(), gomock.Any(), gomock.Any(),
				).Return(&domain.AccountV2{
					AccountV2: &accountproto.AccountV2{
						Email:            "email",
						OrganizationRole: accountproto.AccountV2_Role_Organization_UNASSIGNED,
						EnvironmentRoles: []*accountproto.AccountV2_EnvironmentRole{
							{
								EnvironmentId: "ns0",
								Role:          accountproto.AccountV2_Role_Environment_UNASSIGNED,
							},
						},
					},
				}, nil).AnyTimes()
			},
			req: &accountproto.GetAPIKeyRequest{Id: "id", EnvironmentNamespace: "ns0"},
			getExpectedErr: func(localizer locale.Localizer) error {
				return createError(localizer, statusPermissionDenied, localizer.MustLocalize(locale.PermissionDenied))
			},
		},
```

Reply:
Looks good!